src/common/meta/src/ddl/create_logical_tables/update_metadata.rs

```diff
@@ -81,6 +81,10 @@ impl CreateLogicalTablesProcedure {
 
     pub(crate) async fn create_logical_tables_metadata(&mut self) -> Result<Vec<TableId>> {
         let remaining_tasks = self.data.remaining_tasks();
+        if remaining_tasks.is_empty() {
```

Comment:
It can't be empty because we've already checked it here.:https://github.com/GreptimeTeam/greptimedb/blob/e88465840d2e11bf65eb4f3907aaf95ee04c0878/src/common/meta/src/ddl/create_logical_tables.rs#L100